core/services/vrf/v2/integration_helpers_test.go

```diff
@@ -343,9 +343,11 @@ func testMultipleConsumersNeedTrustedBHS(
 	})
 
 	// Whitelist vrf key for trusted BHS.
-	_, err := uni.trustedBhsContract.SetWhitelist(uni.neil, bhsKeyAddresses)
-	require.NoError(t, err)
-	uni.backend.Commit()
+	{
```

Comment:
Why do we want to scope these lines differently?

Reply:
Just trying to avoid shadowing this `err` var later on. Seemed cleaner than renaming all those to `err2` in this case :shrug: